src/Device/MultipleDevices.cpp

```diff
@@ -106,11 +106,14 @@ MultipleDevices::PutStandbyFrequency(RadioFrequency frequency,
     i->PutStandbyFrequency(frequency, name, env);
 }
 
-void
+bool
 MultipleDevices::ExchangeRadioFrequencies(OperationEnvironment &env)
 {
-  for (DeviceDescriptor *i : devices)
-    i->ExchangeRadioFrequencies(env);
+  for (DeviceDescriptor *i : devices) {
+    return i->ExchangeRadioFrequencies(env);
```

Comment:
This calls only the first device and ignores the others.
Have you ever tested your code?

Reply:
Agreed, that doesn't make sense at all.